[All Hosts] (manifest) reorg the manifest part of the TOC (#4727)
OfficeDev/office-js-docs-pr: docs/includes/manifest-guidance.md

> [!TIP]
> If you'll be testing your add-in across multiple environments (for example, in development, staging, demo, etc.), we recommend that you maintain a different manifest file for each environment. In each manifest file, you can:
> 
> - Specify the URLs that correspond to the environment.

> > - Customize metadata values so that end users will be able to identify a sideloaded add-in's corresponding environment. For example:

Reviewer: ```suggestion
> - Customize metadata values so that end users are able to identify a sideloaded add-in's corresponding environment. For example:
```
